src/API.js

```diff
@@ -1,48 +1,19 @@
-import axios from "axios"
+import axios from 'axios'
 
 const API = {
-    SERWER: "http://localhost:3000",
-    TOKEN: "0123456789",
+  baseHeaders: {
```

Comment:
For the API.js file please add departmentApi which will embed the API.js logic within

Reply:
do you mean to create another file called departmentApi?
what's sense to do it?

Reply:
you need to use a composition
Since in future you will need to add endpoints for employees. Where will you put new endpoints? :)

Reply:
well, i'll add methods for employee ;)
![image](https://user-images.githubusercontent.com/12400717/160398918-18005cc5-2528-4021-9095-d3e723a2ba04.png)